### ISSUE TITLE
`priority` should order columns in an index table too

### ISSUE THREAD

**Issue by srobbin** [vapid/vapid#144]
**Is your feature request related to a problem? Please describe.**
The columns in an index table (see below) are ordered by their appearance in the HTML. Since the `priority` attribute allows you to specify the order of input fields in a form, it should probably also dictate the order of columns in that table.

**Describe the solution you'd like**
Modify the index table logic, so it looks for field priorities.

**Additional context**

![9ee3504e5ae006ada71dc85ef7b728eec5d0d651_2_690x248](https://user-images.githubusercontent.com/44719/58390574-7f3f1100-7ff7-11e9-88cb-e4e3da03212b.png)

**Comment by jeremieemk** [comment]
Are you planning on fixing this one? Thanks

**Comment by jeremieemk** [comment]
Hey great work with Vapid, i have the same issue for a client's project and i was wondering if there is a schedule or a workaround for this issue?

**Comment by walidvb** [comment]
@jeremieemk if the columns are shown in the order of appearance in the html, does using a 'ghost' version work? 

```
{{sections projects multiple=true}}
 <div data-ghost="{{fieldIWandInTheColumns}}">

 </div>
{{/section}}
```

**Comment by srobbin** [comment]
Just published a fix for this in v0.9.1. The section index table should now respect the `priority` attribute. To update to the latest version:

```
npm install -g @vapid/cli
```
